include/openenclave/bits/log-common.h

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef OE_LOG_COMMON_H
+#define OE_LOG_COMMON_H
+
+enum { LOG_TRACE = 0, LOG_DEBUG, LOG_INFO, LOG_WARN, LOG_ERROR };
```

Comment:
Our enums have the OE_ prefix.
Also all enums have a max value declared to coerce the underlying type of the enum.
See oe_result_t